tests/all/cli_tests.rs

```diff
@@ -921,7 +921,7 @@ fn table_growth_failure2() -> Result<()> {
     assert!(!output.status.success());
     let stderr = String::from_utf8_lossy(&output.stderr);
     assert!(
-        stderr.contains("forcing a table growth failure to be a trap"),
+        stderr.contains("forcing trap when growing table to"),
```

Comment:
Do you know why this error changed? Is that perhaps a cast gone wrong somewhere?